gateway-service-knoxtoken/src/main/java/org/apache/knox/gateway/service/knoxtoken/TokenResource.java

```diff
@@ -300,6 +301,16 @@ public void init() throws AliasServiceException, ServiceLifecycleException, KeyL
       tokenMAC = new TokenMAC(gatewayConfig.getKnoxTokenHashAlgorithm(), aliasService.getPasswordFromAliasForGateway(TokenMAC.KNOX_TOKEN_HASH_KEY_ALIAS_NAME));
 
       tokenLimitPerUser = gatewayConfig.getMaximumNumberOfTokensPerUser();
```

Comment:
Typically, topology-level config overrides gateway-level config. This appears to be the opposite.